src/smart-components/workflow/workflows.js

```diff
@@ -45,10 +47,17 @@ const Workflows = ({ fetchRbacGroups, fetchWorkflows, workflows, pagination, his
         title: 'Delete',
         style: { color: 'var(--pf-global--danger-color--100)'	},
         onClick: (event, rowId, workflow) =>
-          history.push(`/workflows/remove/${workflow.id}`)
+          history.push({ pathname: '/workflows/remove-workflow',
+            state: { checkedWorkflows: [ workflow.id ]}
```

Comment:
I am a bit confused by this change. Is there any reason why we cannot react the id from `match.params`?

Reply:
Moved the cheked ids to props